browser/src/ConstraintTable/GnomadNonCodingConstraintTableVariant.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { BaseTable, TooltipAnchor, TooltipHint } from '@gnomad/ui'
+
+import Link from '../Link'
+
+import { renderRoundedNumber } from './constraintMetrics'
+
+const Table = styled(BaseTable)`
+  width: 100%;
+  margin-top: 1rem;
+
+  @media (max-width: 600px) {
+    td,
+    th {
+      padding-right: 10px;
+
+      /* Drop sparkline column */
+      &:nth-child(5) {
+        display: none;
+      }
+    }
+  }
+`
+
+const ViewSurroundingRegion = styled.div`
+  margin-top: 1rem;
+`
+
+type NonCodingConstraint = {
+  start: number
+  stop: number
+  possible: number
+  observed: number
+  expected: number
+  oe: number
+  z: number
+}
+
+type Props = {
+  variantId: string
+  chrom: string
+  nonCodingConstraint: NonCodingConstraint
+}
+
+const GnomadNonCodingConstraintTableVariant = ({
+  variantId,
+  chrom,
+  nonCodingConstraint,
+}: Props) => {
+  if (nonCodingConstraint == null) {
+    return <>This variant does not have non coding constraint data for the surrounding region.</>
+  }
+
+  const variantLocation = parseInt(variantId.split('-')[1], 10)
+  // TODO:(rgrant) do some checks if this goes over the max region, or under
+  const surroundingLocation = `${chrom}-${variantLocation - 20000}-${variantLocation + 20000}`
```

Comment:
This may also be a good use case for numeric separators (e.g. `20_000`, `200_000`) to make them a bit easier to read

see: https://mariusschulz.com/blog/numeric-separators-in-typescript